js/packages/mobile-wallet-adapter-protocol-web3js/src/transact.ts

```diff
@@ -158,3 +178,18 @@ export async function transact<TReturn>(
     };
     return await baseTransact(augmentedCallback, config);
 }
+
+function validateSecureEndpointSpecificURI(walletUriBase?: string) {
```

Comment:
Nullability will kill us all. Prefer: make this argument required, then just don't call this method if you don't have a base URI.

Reply:
Done